app/src/pages/Play/YourTurnContent.tsx

```diff
@@ -177,24 +205,27 @@ function YourTurnContent(props: {
                       <Box>
                         <GreenCheckbox
                           checked={
-                            shownCardsInActiveTurn.get(cardId) ===
-                            ShownCardStatus.Completed
+                            shownCardsInActiveTurn.get(cardId)?.status ===
+                            ShownCardStatus.Complete
                           }
                           onChange={({ target: { checked } }) => {
-                            setShownCardsInActiveTurn(
-                              new Map(
-                                shownCardsInActiveTurn.set(
-                                  cardId,
-                                  checked
-                                    ? ShownCardStatus.Completed
-                                    : ShownCardStatus.Incomplete
+                            const shownCard = shownCardsInActiveTurn.get(cardId)
+                            if (shownCard) {
+                              setShownCardsInActiveTurn(
+                                new Map(
+                                  shownCardsInActiveTurn.set(cardId, {
+                                    ...shownCard,
+                                    status: checked
+                                      ? ShownCardStatus.Complete
+                                      : ShownCardStatus.Incomplete
+                                  })
                                 )
```

Comment:
review the cards

the status could be more sophisticated here to toggle back to Skipped.. but will adjust soon in follow up PRs.